base/cvd/cuttlefish/host/commands/cvd/selector/selector_common_parser.h

```diff
@@ -30,58 +26,18 @@
 namespace cuttlefish {
 namespace selector {
 
-class SelectorCommonParser {
- public:
-  // parses common selector options, and drop the used selector_args
-  static Result<SelectorCommonParser> Parse(cvd_common::Args& selector_args,
-                                            const cvd_common::Envs& envs);
-
-  std::optional<std::string> GroupName() const { return group_name_; }
-
-  std::optional<std::vector<std::string>> PerInstanceNames() const {
-    return instance_names_;
+struct SelectorOptions {
```

Comment:
Is the plan to combine this with the `Query` / `Queries` types? They seem like semantically related concepts, like `Query` can be a filter for an instance id field.

Can this use `Query` already or hold `Query` types? I see there is a conversion in `group_selector.cpp` and `instance_selector.cpp` already.

Reply:
I'll probably end up merging this, `Queries` and `FindParam` together, but there are still some details I want to polish first.